draft-ietf-tls-tls13.md

```diff
@@ -2382,12 +2382,13 @@ The "extension_data" field of this extension contains an
 
 %%% Key Exchange Messages
 
+       struct {} Empty;
```

Comment:
Again, the specification does say that the presentation language is casually defined, I voice my concerns on this :)

(also I think there is another // empty somewhere in the spec)